examples/calculate_rice_score.py

```diff
@@ -0,0 +1,126 @@
+# Reference: https://www.intercom.com/blog/rice-simple-prioritization-for-product-managers/
+from python_http_client import Client
+import os
+import json
+from datetime import datetime
+from email.utils import parsedate_tz, mktime_tz
+
+client = Client(host="http://{}".format(os.environ.get('DX_IP')))
+
+def calculate_rice_reach(task):
+    # Last updated May 2019
+    # https://docs.google.com/spreadsheets/d/1wNImudaEewPijd9EcgsHiEXbermh4amyug-2tTfyOWQ/edit#gid=2047355119
+    rice_reach = 0
+    if task['language'] == 'python':
+        rice_reach = 7744
```

Comment:
Hmm, should this be based on the repo name since we'll be working with Twilio helper libs, too?

Reply:
That's a good point, I'll update this.